config/database.yml

```diff
@@ -66,6 +66,8 @@ test:
   <<: *default
   url: <%= ENV['DATABASE_URL_TEST'] || ENV['DATABASE_URL'] || ''%>
   database: PracticalDeveloper_test<%= ENV['TEST_ENV_NUMBER'] %>
+  variables:
+      statement_timeout: <%= ENV['STATEMENT_TIMEOUT'] || 10_000 %>
```

Comment:
**suggestion (non-blocking):** Maybe we should use `fetch` more:

```suggestion
      statement_timeout: <%= ENV.fetch('STATEMENT_TIMEOUT', 10_000) %>
```

Note: either version will return either a string (if the env var is set) or integers. It's not uncommon to do `ENV.fetch('STATEMENT_TIMEOUT', 10_000).to_i)` in cases like this, since it's just returns the number for integers:

> Since int is already an Integer, returns self.